test/test_EKF_flow.cpp

```diff
@@ -89,8 +89,8 @@ TEST_F(EkfFlowTest, resetToFlowVelocityInAir)
 	const Vector2f simulated_horz_velocity(0.5f, -0.2f);
 	flowSample flow_sample = _sensor_simulator._flow.dataAtRest();
 	flow_sample.flow_xy_rad =
-		Vector2f(- simulated_horz_velocity(1) * flow_sample.dt / estimated_distance_to_ground,
-			   simulated_horz_velocity(0) * flow_sample.dt / estimated_distance_to_ground);
+		Vector2f( simulated_horz_velocity(1) * flow_sample.dt / estimated_distance_to_ground,
```

Comment:
Do you know why this was passing the test before even tough I got the signs wrong.

Reply:
Yes, because the test was just for the reset counter. There is still a bug in the flow reset that always initializes with zero. The value isn't important.